services/_shared/adapters/pg_install_store.py

```
@@ -286,14 +290,29 @@ def set_repo_config(
+        and flag not in _ALLOWED_STRING_FLAGS
     }
     if non_bool:
         raise TypeError(
             f"set_repo_config() persona flags must be bool or None; got {non_bool!r}"
         )
+    # Voice entitlement gate (#288 paid voice pack): yoda is only for paid installs
+    if "elder_voice" in persona_flags and persona_flags["elder_voice"] == "yoda":
+        from adapters.install_store import get_installation
+        install = get_installation(install_id)
+        if not install:
+            raise ValueError(f"Installation {install_id} not found")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)